omero/sysadmins/unix/server-centos6.txt

```diff
@@ -28,113 +28,75 @@ consequence, it is necessary to upgrade to Python 2.7 in order to
-    yum install python27
-
-Note that ``epel-release`` will be pulled as a dependency.
-
-To use the newly installed version of Python, it is preferrable to
-install the various dependencies required to run the
-OMERO.server in a virtual environment.
-
-If virtualenv is **not** already installed, install it using pip::
-
-	yum install python27-pip
-	pip2.7 install virtualenv
-
-Development packages
-^^^^^^^^^^^^^^^^^^^^
-
-The following sections require a number of libraries and a compiler.
-Run the following commands to install them::
-
-    yum groupinstall "Development Tools"
-    # Python module dependencies
-    yum install python27-devel hdf5-devel freetype-devel libjpeg-devel libpng-devel
-
-
 Ice 3.5
 ^^^^^^^
 
 The RPM packages provided by ZeroC for CentOS 6 use the Python 2.6
 provided by the system. OMERO will need an Ice build which uses
 Python 2.7. A version of Ice 3.5 built against Python 2.7 installed
 above is available at http://downloads.openmicroscopy.org/ice/experimental.
+Ice will be installed into :file:`/opt`.
 
-The following steps will install Ice into :file:`/opt`::
-
-    # Download and install needed Ice build dependencies (db and mcpp)
-    curl -o /etc/yum.repos.d/zeroc-ice-el6.repo http://download.zeroc.com/Ice/3.5/el6/zeroc-ice-el6.repo
-    mkdir /tmp/ice-download
-    cd /tmp/ice-download
-    wget http://downloads.openmicroscopy.org/ice/experimental/Ice-3.5.1-b1-centos6-iuspy27-x86_64.tar.gz
-    yum install db53 db53-devel db53-utils mcpp-devel 
-    tar -zxvf /tmp/ice-download/Ice-3.5.1-b1-centos6-iuspy27-x86_64.tar.gz
-    mv Ice-3.5.1-b1-centos6-iuspy27-x86_64 /opt/Ice-3.5.1
-    # make path to Ice globally accessible
-    # if globally set, there is no need to export LD_LIBRARY_PATH
-    echo /opt/Ice-3.5.1/lib64 > /etc/ld.so.conf.d/ice-x86_64.conf
-    ldconfig
-
-Prerequisites
--------------
-
-This is an example walkthrough for installing OMERO on CentOS 6 with Python 2.7 using
-a dedicated system user.
-You can use this as a guide for setting up your own test server. For production use you should also read the pages listed under :ref:`index-optimizing-server`.
-
-For convenience in this walkthrough the main OMERO configuration options have
-been defined as environment variables. When following this walkthrough you can
-either use your own values, or alternatively source the following file :download:`settings.env <walkthrough/settings.env>`:
+Ice 3.6
+^^^^^^^
+With Ice 3.6, the Python bindings are provided separately.
+This allows to install the RPM packages provided by ZeroC for CentOS 6.
+Then run ``pip install zeroc-ice`` to install the Ice Python bindings if
+your package manager does not provide the Ice python packages.
+See `Using the Python Distribution
+<https://doc.zeroc.com/display/Ice36/Using+the+Python+Distribution>`__
+for further details.
 
-.. literalinclude:: walkthrough/settings.env
-   :start-after: Substitute
 
+**The following steps are run as root.**
```

Comment:
We need a new subheading here so this section doesn't all fall within the 'Ice 3.6' section, I suggest 'Installing Prerequisites'